docs/src/snap/howto/install/install-offline.md

```diff
@@ -0,0 +1,272 @@
+# Installing Canonical Kubernetes in airgapped environments
+
+There are situations where it is necessary or desirable to run Canonical 
+Kubernetes on a machine that is not connected to the internet. 
+Based on different degrees of separation from the network,
+different solutions are offered to accomplish this goal.
+This guide documents any necessary extra preparation for airgap deployments,
+as well the steps that are needed to successfully deploy Canonical Kubernetes
+in such environments.
+
+## Prepare for Deployment
+
+In preparation for the offline deployment you will download the Canonical
+Kubernetes snap, fulfill the networking requirements based on your scenario and
+handle images for workloads and Canonical Kubernetes features.
+
+### Prep 1: Download the Canonical Kubernetes snap
+
+From a machine with access to the internet download the following:
+
+```bash
+sudo snap download k8s --channel 1.30-classic/beta --basename k8s
```

Comment:
The `1` is weirdly rendered in the final doc. Not sure how to avoid that, @evilnick maybe?

Reply:
I think adding `bash` does something funny to the formatting?

Reply:
bash is only for bash **scripts** , not cli commands. Just leave off the type